### DISCUSSION TITLE
[Suggestion] Different Color Hit Splat for Different Zero Hits

### DISCUSSION THREAD

**Discussion by petertalbanese** [runelite/runelite#15285] [Q&A] [answered]
A simple plugin which recolors your hit splat for a rolled 0 (e.g. you hit your target but rolled 0 damage) vs missed 0.

I've never made a Runelite Plugin before, but can give this a try as it seems pretty simple. Would this be against any game rules?

**Comment by Enriath** [comment, marked as answer]
It is not possible for the client to know whether a 0 hit actually hit or not. This is something Jagex would have to implement.

**Reply by petertalbanese** [reply]
Ah, unfortunate. Thanks for the reply!
